src/dev-app/theme.scss

```diff
@@ -63,8 +65,8 @@ $dark-theme: mat-dark-theme((
 // `.demo-unicorn-dark-theme` will be affected by this alternate dark theme instead of the
 // default theme.
 .demo-unicorn-dark-theme {
-  @include angular-material-theme($dark-theme);
-  @include angular-material-mdc-theme($dark-theme);
+  @include angular-material-color($dark-theme);
```

Comment:
does this issue also affect the individual mixins (e.g. mat-button-theme)? if so should we change the experimental ones below? I know they're experimental, but someone might look at this as an example and I don't want it to confuse them

Reply:
It definitely could affect the individual mixins too. My thinking was that for these it's probably not needed as individual mixins are usually consumed in more advanced applications, and because it will result in a lot of bloat.

I don't feel strongly about it though. Happy to change if desired.

Reply:
Can you add some guidance on how to handle the individual mixins to the documentation then? I assume it's something like explicitly passing null for the density map?

Reply:
Yeah, either that, or using the actual `-color` mixin and extracting the color config from the theme. Both somehow feel not very intuitive for me.

I think extracting the color config conceptually makes the most sense and API wise isn't too bad (if we explain well how themes work).

Reply:
Updated the doc. Please have another look. I'm still unsure whether we want to add the automatic reporting if duplicate styles are generated from the individual theme mixins. I could definitely see us doing that. What do you personally think? I think we can discuss this in the next meeting and send a follow-up.

Reply:
I think its more important to include them for the individual mixins than the combined mixin. The people who are using the combined mixin probably don't care about size as much anyways. But yeah, lets add an item to the agenda and discuss

Reply:
Oh, that's a good point. I didn't think of it that way. I strongly agree with what you said. I'll make the changes for adding it to individual mixins.

Reply:
@mmalerba Done. I added a lint rule for it too. It already catched a few cases in `dev-app/theme.scss` :+1:.

Also, I added it to the non-experimental mixins too. If we decide to not want to report a warning (shouldn't be breaking), we easily filter out `mat-*` components.